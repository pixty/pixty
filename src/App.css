.App-logo {
  animation: App-logo-spin infinite 2s linear;
  height: 40px;
}

.App-header {
  background: #333;
  overflow: hidden;
  padding: 0px;
  color: white;
}

.Snapshot--size {
  width: 330px;
  min-height: 600px;
}

@media only screen and (max-device-width: 480px) {
  .Snapshot--size {
    width: 320px;
    height: 180px;
  }
}

.App-intro {
  font-size: large;
}

.Scene {
  padding: 0px;
  box-shadow: 2px 2px 15px rgba(0,0,0,0.6)
}

.Scene--size {
  height: 80px;
}


.Face {
  position: absolute;
  top: 0px;
  left: 0px;
  bottom: 0px;
  right: 0px;
  border: 2px solid orange;
  border-radius: 5px;
}

@keyframes App-logo-spin {
  from { transform: rotate(0deg); }
  to { transform: rotate(360deg); }
}
