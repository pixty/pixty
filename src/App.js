import React, { Component } from 'react';
import './App.css';
import { push } from 'react-router-redux';
import { connect } from 'react-redux';
import Scene from './containers/Scene';
import LoadingBar from 'react-redux-loading-bar';
import { mainColor } from './components/styled/Colors';


class App extends Component {
  render() {
    return (
      <div className="App">
        <LoadingBar style={{ backgroundColor: {mainColor}, height: '2px' }} />
        <Scene />
      </div>
    );
  }
}

export default connect()(App);
