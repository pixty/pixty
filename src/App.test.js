import React from 'react'
import { MemoryRouter } from 'react-router-dom'
import { Provider } from 'react-redux'
import sinon from 'sinon'
import configureStore from 'redux-mock-store'
import App from './App'

const middlewares = []
const mockStore = configureStore(middlewares)

it('App renders without crashing', () => {

  const initialState = {
    entities: {
      scenes: {}
    },
    router: {},
    loadingBar: 0,
    selectedPerson: {}
  }

  const store = mockStore(initialState)  
  const dispatchSpy = sinon.spy()
  store.dispatch = dispatchSpy

  const app = render(
    <Provider store={store}>
      <MemoryRouter>
        <App />
      </MemoryRouter>
    </Provider>
  )
  expect(app).toMatchSnapshot()
})
