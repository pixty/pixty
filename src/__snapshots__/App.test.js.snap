// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders without crashing 1`] = `
<div
  class="App"
>
  <div />
  <div
    style="position:absolute;top:0;left:0;right:0;width:100%;height:100%;overflow:hidden;"
  >
    <div />
    <div
      class="react-draggable sc-ifAKCX eyOSGJ"
      style="touch-action:none;position:absolute;width:142px;height:80px;border-radius:6px;transform:translate(100px,100px);"
    >
      <img
        alt="Face"
        class="Scene--size"
        src="http://pixty.io/assets/snapshots/rest0103.png"
        style="border-radius:5px;"
      />
    </div>
    <div
      class="sc-iwsKbI cAHoiV"
    >
      <div
        style="margin:0px;display:flex;align-items:center;justify-content:center;"
      >
        <div
          style="margin:0px 15px;width:auto;flex-grow:0;"
        >
          <div
            style="float:left;"
          >
            <img
              src="http://www.gravatar.com/avatar/6604f6e418d8db645955aa2626f7b309?s=60&d=identicon"
              style="width:30px;border-radius:30px;margin-top:10px;"
            />
          </div>
          <div
            style="float:left;color:#ccc;margin-top:15px;margin-left:10px;font-size:14px;font-weight:bold;"
          />
        </div>
        <div
          style="margin:0px 15px;width:auto;flex-grow:1;"
        >
          <div
            style="float:left;margin-left:15px;"
          >
            <div
              style="position:relative;"
            >
              <input
                class="sc-bdVaJa jCnrIS"
                src="/images/camera.svg"
                type="image"
                width="25px"
              />
              <div
                class="_drop_down sc-dnqmqq lhDpnN"
                style="visibility:hidden;opacity:0;"
              >
                <ul>
                  <li />
                </ul>
              </div>
            </div>
          </div>
          <div
            style="float:left;margin-bottom:7px;margin-left:10px;"
          >
            <span
              style="color:#888;font-size:13px;font-weight:200;"
            />
          </div>
          <div
            style="float:left;margin-left:15px;"
          >
            <input
              class="sc-bdVaJa jCnrIS"
              src="/images/eye.svg"
              type="image"
              width="25px"
            />
          </div>
        </div>
        <div
          style="width:25px;margin-right:15px;"
        >
          <div
            style="position:relative;"
          >
            <input
              class="sc-bdVaJa jCnrIS"
              src="/images/settings.svg"
              type="image"
              width="25px"
            />
            <div
              class="_drop_down sc-dnqmqq eZMwGw"
              style="visibility:hidden;opacity:0;"
            >
              <ul>
                <li>
                  <a
                    href="/analytics"
                  >
                    Analytics
                  </a>
                </li>
                <hr />
                <li>
                  Settings
                </li>
                <li>
                  <a
                    href="/logout"
                  >
                    Logout
                  </a>
                </li>
              </ul>
            </div>
          </div>
        </div>
      </div>
      <div
        style="position:absolute;top:50%;left:50%;"
      >
        <div>
          <div
            class="sc-htpNat hhAasZ"
          >
            identifying process
          </div>
          <svg
            class="sc-bxivhb jecG"
            opacity="0.5"
            viewbox="0 0 50 50"
          >
            <circle
              class="path"
              cx="25"
              cy="25"
              fill="none"
              r="20"
              stroke-width="1"
            />
          </svg>
        </div>
      </div>
    </div>
    <div
      class="sc-gZMcBi coPbFD"
      opacity="0"
      width="0px"
    >
      <div
        style="padding:10px;"
      />
    </div>
  </div>
</div>
`;
